example/rollup/lite.pl

```diff
@@ -1,20 +1,37 @@
 #!/usr/bin/env perl
 use Mojolicious::Lite;
-use FindBin;
-BEGIN { unshift @INC, "$FindBin::Bin/../../lib" }
+use Mojo::File 'curfile';
+require lib;
 
-plugin Webpack =>
-  {process => ['svelte'], dependencies => {core => 'rollup', svelte => [qw(rollup-plugin-svelte svelte)]}};
+$ENV{MOJO_WEBPACK_CONFIG}  = "@{[curfile->sibling('rollup.config.js')]}";
+$ENV{MOJO_WEBPACK_DEBUG}   = 1;
+$ENV{MOJO_WEBPACK_VERBOSE} = 0;
+$ENV{MOJO_WEBPACK_LAZY}    = 0;
```

Comment:
Can you explain why all of these environment variables are required now?

Reply:
`MOJO_WEBPACK_CONFIG` is required otherwise `Mojolicious::Plugin::Webpack` decides it is a `webpack` project rather than `rollup`. Others are removed as they were added in haste.